src/api.raml

```diff
@@ -832,6 +832,44 @@ traits:
                                 description: |
                                     Demonstration of the list being empty (the first page will always return a valid result).
                                 value: !include samples/digest-list/v1/empty.json
+    post:
+        description: |
+            Add a new digest.
+        body:
+            application/vnd.elife.digest+json;version=1:
+                schema: !include ../dist/model/digest.v1.json
+                examples:
+                    minimum:
+                        displayName: Minimum
+                        value: !include samples/digest/v1/minimum.json
+                    complete:
+                        displayName: Complete
+                        value: !include samples/digest/v1/complete.json
+        responses:
+            200:
+                description: |
+                    Digest just created.
+                body:
+                    application/vnd.elife.digest+json;version=1:
+                        schema: !include ../dist/model/digest.v1.json
+                        examples:
+                            minimum:
+                                displayName: Minimum
+                                value: !include samples/digest/v1/minimum.json
+                            complete:
+                                displayName: Complete
+                                value: !include samples/digest/v1/complete.json
+            400:
+                description: Missing or invalid fields for digest.
+            403:
+                description: Not authorized to add a digest.
+                #body:
```

Comment:
would add, but not implemented on 403?

Reply:
Need to include type, schema and example like https://github.com/elifesciences/api-raml/blob/534c9dde2965337a91f5a41b0b9744d644e87249/src/api.raml#L84-L103.